<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="menuWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Read poly object</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="drawFile">
               <property name="text">
                <string>Draw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseFile">
               <property name="text">
                <string>browse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QVBoxLayout">
             <property name="leftMargin">
              <number>52</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>3D Objects</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="drawButton">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Draw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="shapes">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Cylinder</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sphere</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Cone</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Cube</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arrow</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Disk</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="infoMainWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>10</y>
               <width>171</width>
               <height>31</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Shape Info</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QVTKOpenGLStereoWidget" name="openGLWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLStereoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglstereowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
